src/components/gameScreen/gameScreen.container.js

```diff
@@ -71,40 +108,22 @@ export const GameScreenContainer = () => {
   const timerless = () => {
     return <View style={styles.block}/>
   }
-
+  const bookmarkSetter = () => {
+    api.setBookmark({
+      question: currentQuestion,
+      help: filteredQuestion[0].help,
+      rightAnswer: filteredQuestion[0].rightAnswer,
+    }).then(setBookmarkAddingStatus)
+      .catch( (e) => console.log('bookmark add error', e));
+  }
   const nextButton = () => {
     return <TouchableOpacity style={[styles.nextButton, { backgroundColor: mainColor }]} onPress={() => handleNextQuestion(currentScore, true)}>
       <Text style={ styles.buttonText }>Next</Text>
     </TouchableOpacity>
   }
 
-  const handleNextQuestion = (updatedScore, isNextButton) => {
-    setCurrentScore(updatedScore);
-    if ((chosenMode === 'Hard') || (isNextButton)) {
-      navigation.dispatch(
-        CommonActions.reset({
-          index: 1,
-          routes: [
-            { name: 'QuizGame' },
-            {
-              name: 'Game',
-              params: {
-                questionNumber: questionNumber + 1,
-                navigation: navigation,
-                category: category,
-                score: updatedScore,
-                chosenMode: chosenMode,
-                mainColor: mainColor
-              }
-            }
-          ]
-        })
-      )
-    }
-
-  }
   const renderHelpDialog = () => {
-    return <HelpDialog setShowHelpDialog={setShowHelpDialog} mainColor={ mainColor }/>
+    return <HelpDialog setShowHelpDialog={setShowHelpDialog} mainColor={ mainColor } theory={filteredQuestion[0].help}/>
   }
   const answersAnimation = useAnimatedStyle(() => {
```

Comment:
why do you use react-native-reanimated and Animated from react native in the same project?

Reply:
use only react-native-reanimated